HomeTask5/src/Controller.js

```diff
@@ -0,0 +1,104 @@
+export default class Controller {
+    constructor(model, view) {
+        this._model = model;
+        this._view = view;
+    }
+
+    SelectCategory(category) {
```

Comment:
rewrite the same like selectCategory = (category) => {
}

all members of a class should be in camel case and if there are any constants they will be in upper case